### PR TITLE
Adds the english ul element

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_heart:  Ready to go! Same as other elements I added.Thanks!

### CODE REVIEW

**Comment by savance-ford** [comment]
:+1:  Looks good to me!

File: elements/ul.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <p>Unordered lists will start with bullets as they are not in order and have no numbering system to keep them in a specific order.</p>
+    </div>
+  </div>
+  <pre class="code-example">
+    <code>
+      <strong>&lt;ul&gt;</strong>
+        &lt;li&gt;Computers&lt;/li&gt;
+        &lt;li&gt;Power Cords&lt;/li&gt;
+        &lt;li&gt;Keyboards&lt;/li&gt;
+      <strong>&lt;/ul&gt;
```
Comment (rylnd) [inline elements/ul.html]: Missing the closing `</strong>` again

Comment (kathyoleary) [inline elements/ul.html]: I'm just gonna smack myself silly

File: elements/ul.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Used to group a list of items where the order that they are in is not important.  The bullet style can be changed to a wide variety of objects from dots to dashes to squares or even pictures.</p>
```
Comment (rylnd) [inline elements/ul.html]: The clause "that they are in" is redundant and kinda just noise. I would remove it. Maybe just "... a list of items whose order is not important."

**Comment by rylnd** [comment]
:+1: once comments are addressed

File: elements/ul.html (hunk 1 above)
Comment (kathyoleary) [inline elements/ul.html]: ok fixed

**Pull request merged**